### PR TITLE
Implement finance.yml as variables

### PR DESCRIPTION
**Pull request by MylesJarvis**
### Trello card

https://trello.com/c/2WHavuTv

### Context

We are centralising our monetary values into one place. This PR does this for finance related values.

### Guidance to review

Pages where we have changes:

[ ] https://get-into-teaching-app-review-4026.test.teacherservices.cloud/funding-and-support/if-youre-a-parent-or-carer
[ ] https://get-into-teaching-app-review-4026.test.teacherservices.cloud/funding-and-support/tuition-fee-and-maintenance-loans
[ ] https://get-into-teaching-app-review-4026.test.teacherservices.cloud/landing/how-to-fund-your-teacher-training
[ ] https://get-into-teaching-app-review-4026.test.teacherservices.cloud/non-uk-teachers/non-uk-qualifications

### Pre-election period restrictions

* [ ] This change is permitted within the constraints of the [pre-election period guidelines](https://www.gov.uk/government/publications/election-guidance-for-civil-servants/general-election-guidance-2024-guidance-for-civil-servants-html#publishing-content-online-)

### CODE REVIEW

File: app/views/content/funding-and-support/if-youre-a-parent-or-carer.md [hunk 1]
```diff
@@ -71,15 +71,15 @@ To be eligible, you need to:
 
 The Parents’ Learning Allowance could help you with your learning costs if you’re a full-time student with children.
 
-You could be eligible for between £50 and £1,963 a year, depending on your household income.
+You could be eligible for between $finance_parentslearningallowance_lowpayment$ and $finance_parentslearningallowance_maxpayment$ a year, depending on your household income.
```
Comment (Sarah-DfE) [inline app/views/content/funding-and-support/if-youre-a-parent-or-carer.md:74]: finance_parentslearningallowance_low (although does it make more sense that this ends in min if the other one is max?)

finance_parentslearningallowance_max

File: app/views/content/funding-and-support/if-youre-a-parent-or-carer.md [hunk 2]
```diff
@@ -71,15 +71,15 @@ To be eligible, you need to:
...
 
 This is usually paid to you in 3 instalments, one at the start of each term.
 
 To be eligible, you need to:
 
 * be doing a full-time undergraduate course or a teacher training course in England
 * have dependent children
-* have a household income of less than £18,835.98 a year
+* have a household income of less than $finance_parentslearningallowance_incomethreshold$ a year
```
Comment (Sarah-DfE) [inline app/views/content/funding-and-support/if-youre-a-parent-or-carer.md:82]: Shorten to
finance_parentslearningallowance_maxincome

File: app/views/content/funding-and-support/tuition-fee-and-maintenance-loans.md [hunk 3]
```diff
@@ -44,19 +44,19 @@ The maximum tuition fee loan for postgraduate teacher training for a UK applican
...
+The maximum amount you may be eligible to borrow is:
 
-* £8,610, if you live at home with your parents
-* £10,227, if you live away from your parents, out of London
-* £13,348, if you live away from your parents, in London
-* £4,327, if you're 60 years or over on the first day of the academic year of your course  
+* $finance_maintenanceloanpaymentvalue_livewithparents$, if you live at home with your parents
+* $finance_maintenanceloanpaymentvalue_awayfromparentsoutsidelondon$, if you live away from your parents, out of London
+* $finance_maintenanceloanpaymentvalue_awayfromparentsinsidelondon$, if you live away from your parents, in London
+* $finance_maintenanceloanpaymentvalue_oversixty$, if you're 60 years or over on the first day of the academic year of your course
```
Comment (Sarah-DfE) [inline app/views/content/funding-and-support/tuition-fee-and-maintenance-loans.md:59]: finance_maintenanceloan_withparents
finance_maintenanceloan_notwithparentsinlondon
finance_maintenanceloan_oversixty

File: app/views/content/landing/how-to-fund-your-teacher-training/_content.html.erb [hunk 4]
```diff
@@ -99,9 +99,9 @@
       <p>You could also be eligible for up to:</p>
 
       <ul>
-        <li>£331.95 a week to support childcare costs for 2 or more children</li>
+        <li><%= v :finance_childcaregrantweeklypayment_twoormorechildren %> a week to support childcare costs for 2 or more children</li>
         <li><%= v :finance_adultdependantsgrant_value %> a year if an adult depends on you</li>
```
Comment (Sarah-DfE) [inline app/views/content/landing/how-to-fund-your-teacher-training/_content.html.erb:103]: This isn't showing as a figure in the left column, but showing ok in the review app

**Review by Sarah-DfE — COMMENTED** [review]
@gemmadallmandfe another one for you, lots of potential shortening on this one

File: app/views/content/funding-and-support/if-youre-a-parent-or-carer.md (hunk 2 above)
Comment (Sarah-DfE) [reply app/views/content/funding-and-support/if-youre-a-parent-or-carer.md:82]: finance_parentslearningallowance_income

File: app/views/content/funding-and-support/tuition-fee-and-maintenance-loans.md (hunk 3 above)
Comment (Sarah-DfE) [reply app/views/content/funding-and-support/tuition-fee-and-maintenance-loans.md:59]: $finance_maintenanceloan_awayfromparentsoutsidelondon$

File: config/values/finance.yml [hunk 5]
```diff
@@ -1,19 +1,19 @@
 finance:
-  adultdependantsgrant:
+   adultdependantsgrant:
```
Comment (gemmadallmandfe) [inline config/values/finance.yml:2]: not sure why we are adding a space in front of this one?

File: config/values/finance.yml [hunk 6]
```diff
@@ -1,19 +1,19 @@
...
-    onechild: £193.62
-    twoormorechildren: £331.95
+    one: £193.62
+    twoplus: £331.95
   disabilityadditionalfunding: £26,948
-  maintenanceloanpaymentvalue:
-    liveathomewithparents: £8,610
-    awayfromparentsoutsidelondon: £10,227
-    awayfromparentsinsidelondon: £13,348
+  maintenanceloanvalue:
```
Comment (gemmadallmandfe) [inline config/values/finance.yml:9]: do we even need `value` on the end of this?

File: config/values/finance.yml [hunk 7]
```diff
@@ -1,19 +1,19 @@
 finance:
```
Comment (gemmadallmandfe) [inline config/values/finance.yml:1]: there are a lot build errors on this PR to do with parsing this file - can you help @martyn-w ?

**Review by gemmadallmandfe — COMMENTED** [review]
@Sarah-DfE A couple of comments on the finance PR!

File: config/values/finance.yml (hunk 5 above)
Comment (Sarah-DfE) [reply config/values/finance.yml:2]: Think that was me copying and pasting, will amend

File: config/values/finance.yml (hunk 6 above)
Comment (Sarah-DfE) [reply config/values/finance.yml:9]: Agree, will amend now

File: config/values/finance.yml [hunk 8]
```diff
@@ -1,19 +1,19 @@
...
   parentslearningallowance:
-    lowpayment: £50
-    maxpayment: £1,963
-    incomethreshold: £18,835.98
-  statementofcomparabilitycost: £49.50 plus VAT
\ No newline at end of file
+    min: £50
+    max: £1,963
+    income: £18,835.98
+  statementofcomparabilitycost: £49.50 + VAT
```
Comment (gemmadallmandfe) [inline config/values/finance.yml:19]: @Sarah-DfE I think this is in the wrong config file

This is the cost of a statement of comparability (ie to do with equivalent qualifications) - it is not to do with student finance

I think we should move it to the non-uk config file?

Comment (gemmadallmandfe) [reply config/values/finance.yml:19]: If easier though, we could leave on this PR and move as part of a tidy-up when all the other PRs have gone out?

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me - happy for this to be merged on Thursday

We can move the statement of comparability variable retrospectively through another PR - I'll make a note on the ticket

File: config/values/finance.yml (hunk 8 above)
Comment (Sarah-DfE) [reply config/values/finance.yml:19]: Agree, and good spot!

**Review by Sarah-DfE — APPROVED** [review]
Done a final check and all looks good

**Pull request merged**